src/components/Modal/SearchListModal.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+
+function SearchListModal({ datas, searchText }) {
+  const history = useHistory();
+
+  const result = datas.filter((i, idx) => {
+    return i.title.includes(searchText);
+  });
```

Comment:
```suggestion
  const result = datas.filter(i => i.title.includes(searchText));
```
index 사용안하는데 매개변수 자리 만들어 둘 필요 없어보입니다!
result 보다 직관적인 이름으로 수정해주세요!

Reply:
잃어버린줄 알았는데 검색 관련 코드들이 여기 있었네요... ㅠ....